test/integration/taxon_browsing_test.rb

```diff
@@ -152,6 +153,23 @@ def and_i_can_see_links_to_the_child_taxons_in_an_accordion
     end
   end
 
+  def and_the_accordion_has_tracking_attributes
+    tracked_links = page.all(:css, "a[data-track-category='navAccordionLinkClicked']")
+    tracked_links.size.must_equal @child_taxons.size * search_results.size
+
+    tracked_links.each_with_index do |link, index|
+      section_number = (index / search_results.size).floor + 1
+      item_number = index % search_results.size + 1
+
+      link[:'data-track-action'].must_equal "#{section_number}.#{item_number}"
+      link[:'data-track-label'].must_equal "#{search_results[item_number - 1]['link']}"
+      link[:'data-track-value'].must_equal "#{search_results.size}"
+      link[:'data-track-dimension'].must_equal "#{search_results[item_number - 1]['title']}"
```

Comment:
I wonder whether it would be better to test a couple of links with real values rather than extracting values from `search_results`? The logic in the test is very similar to the logic in the code it's testing.